test_requirements.txt

```diff
@@ -1,2 +1,2 @@
-pytest>=3.3
+pytest>=3.3,<=5.2.2
```

Comment:
`!=5.2.3` might be better.
But in general you could pin it to a fixed version also, e.g. using pip-compile from pip-tools.

Reply:
Thank you. I was afraid that this may still be broken for future updates of pytest, but it looks like they have now resolved this for 5.2.4.